neuron/job.go

```diff
@@ -99,17 +139,22 @@ func (jn *jobNeuron) Trigger(imp spec.Impulse) (spec.Impulse, spec.Neuron, error
 				return
 			}
 
-			response, err := imp.GetState().GetBytes("response")
+			impState, err := imp.GetState("default")
+			if err != nil {
+				fmt.Printf("%#v\n", maskAny(err))
```

Comment:
Now that logging injection is basically solved, we actually need to use the logger everywhere. We should at least replace all `fmt.Printf` calls within the packages.